router/server.go

```diff
@@ -102,6 +103,29 @@ func main() {
 	apiPort := flag.String("api-port", "", "api listen port")
 	flag.Parse()
 
+	if ports := os.Getenv("HTTP_PORTS"); ports != "" {
```

Comment:
Actually, I found the real request: Allow disabling port 80 (#3270). @titanous classified as an enhancement, and this would allow that.

Not sure how we want to handle the "shoot in foot" issue, in that case. Maybe a CLI confirmation for changes which could accomplish it?